### PR TITLE
`lazy_static` replaced by `std::sync::OnceLock`

### PR DESCRIPTION
**Pull request by ThisAccountHasBeenSuspended**
[`lazy_static`](https://crates.io/crates/lazy_static) is no longer needed since [`Rust 1.70.0`](https://releases.rs/docs/1.70.0/)

### Edit
Changes have been made [HERE ](https://github.com/igorlira/dirplayer-rs/pull/30/commits/c2e31c479144ca3c81c8a631db4e950d5145e317) to reduce memory usage and improve performance.

- `String` was changed to `Box<str>` to reduce memory usage.
- The helper functions in `constants.rs` now return a reference instead of a copy which reduces memory usage and improves performance.

### CODE REVIEW

**Comment by igorlira** [comment]
This is great, but there's some conflicts with some other PRs I just merged, can you update this branch?

**Comment by ThisAccountHasBeenSuspended** [comment]
💯

File: vm-rust/src/director/chunks/handler.rs
Comment (ThisAccountHasBeenSuspended) [inline vm-rust/src/director/chunks/handler.rs:1]: The mistake was made [HERE](https://github.com/igorlira/dirplayer-rs/pull/30/commits/8719d370ea8d87cced21e6b36ecf00683baef360#diff-79fe7a17721b868a690ea4d94551dacec0c53d8facce71cfc70df284cf641b60R88) 😬

**Comment by ThisAccountHasBeenSuspended** [comment]
💯

**Review by igorlira — APPROVED** [review]

**Pull request merged**